src/lib/viewer.ts

```diff
@@ -0,0 +1,16 @@
+import {getGraphQLClient} from '../utils/configured-graphql-client'
+import gql from 'graphql-tag'
+
+export async function loadCurrentViewerRoles(token: string) {
+  const query = gql`
+    query viewer {
+      viewer {
```

Comment:
this is an alias server side in gql for current user and generally in egghead-next we've tried to use "viewer" terminology over "user" with varied levels of success

the idea being that a "viewer" isn't necessarily the user but could be support viewing as the user, as a for instance

that's why we have a ViewerProvider etc

Reply:
Figured gql just grabbing the roles array would be quicker than jbuilder any day of the week